changelog/@unreleased/pr-257.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: Skip logging telemetry on liveness, readiness, and health routes.
```

Comment:
Can you update the changelog description to make it more clear what is being changed? Just reading this in the release notes wouldn't be sufficient for me to fully understand the change.

What telemetry is being skipped -- request logs, service logs, something else? Is it being disabled by default or removed completely? What is the motivation of the change? Having some of this context will greatly help understanding the change later.